modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/ClusterStateChangeCommand.java

```diff
@@ -151,6 +169,43 @@ public class ClusterStateChangeCommand extends AbstractCommand<ClusterState> {
         }
     }
 
+    private boolean partialActivationCheck(GridClient client, GridClientConfiguration clientCfg) throws GridClientException {
+        BaselineArguments args = new BaselineArguments.Builder(BaselineSubcommands.COLLECT).build();
+
+        VisorBaselineTaskResult res = executeTaskByNameOnNode(
+                client,
+                VisorBaselineTask.class.getName(),
+                new VisorBaselineTaskArg(
+                        args.getCmd().visorBaselineOperation(),
+                        args.getTopVer(),
+                        args.getConsistentIds()
+                ),
+                coordinatorId(client.compute()),
+                clientCfg
+        );
+
+        Map<String, VisorBaselineNode> baseline = res.getBaseline();
+
+        if (baseline == null) return true;
+
+        for (VisorBaselineNode node : baseline.values()) {
+
+            String constId = node.getConsistentId();
+            VisorBaselineNode srvNode = res.getServers().get(constId);
+
+            if (srvNode == null) offlineNodes.add(constId);
+
+        }
+
+        if (!offlineNodes.isEmpty()) {
+            System.out.println("WARNING: PARTIAL ACTIVATION detected. Baseline has " +
```

Comment:
please use a `log` instance available in the class instead of `System.out` stream, log this message at `warn` level

I also suggest to remove the word `WARNING` as we'll employ `log.warn(...)` method and warning will be included in the logged line automatically. CAPS for PARTIAL ACTIVATION may also be excessive.